packages/cli/pipelines/main.ttl

```diff
@@ -0,0 +1,104 @@
+@base <urn:pipeline:data-cube-curation> .
```

Comment:
It's the IDE plugin which gets rather inconsistent 🙄

Reply:
Any way to disable it?

Reply:
Such negativity 😓. 

But I've just figured it out. The plugin refused to format the triples because it consider groups with common subject to determine what has changed in a file (and leaves unchanged stuff alone).

Check it now

Reply:
Sorry, didn't intend to sound negative

But when editors don't do what we want, there's always the option to do it manually ;)

Reply:
Well, it did turn out, again, the the problem sits between the chair and the keyboard